inbox/api/update.py

```diff
@@ -9,26 +9,28 @@
 # STOPSHIP(emfree): better naming/structure for this module
 
 
-def update_message(message, request_data, db_session):
+def update_message(message, request_data, db_session, optimistic=True):
```

Comment:
Just a preference, but I'd prefer to default optimistic to `None` and `assert(optimistic is not None)` to make sure that this diff isn't overlooking any places this function is called. Defaulting to true could mean we don't find a bug somewhere where the code hasn't been updated and should be passing False.